### ISSUE TITLE
Add UDF parsing (was: no information on BluRay ISOs)

### ISSUE THREAD

**Issue by Feathered-Serpent** [GerbilSoft/rom-properties#344]
Hey there

I know, Rom Properties is mainly there for games, but it also shows information from ISO images from games etc or from Video DVDs, like this one:
![grafik](https://user-images.githubusercontent.com/72162369/178074225-bdf9ca5b-7973-4037-b2b7-d0e90cf3239a.png)

Though this doesn't work on all DVD ISOs and especially it doesn't work at all for BluRay ISOs. Is this because of the file system on BluRays? Because they usually don't have any ISO file system?

**Comment by Masamune3210** [comment]
Yes, if rom-properties understands the contents of the iso, it will try to display its information, if not it just wont show up

**Comment by GerbilSoft** [comment]
Blu-ray discs are normally just UDF without ISO-9660. rom-properties currently doesn't support UDF; it only supports showing if an ISO-9660 disc has UDF in addition.

UDF is on the Todo list, in part because it's needed for PS3 discs.

**Comment by Feathered-Serpent** [comment]
According to Wikipedia "UDF shares the basic volume descriptor format with ISO 9660." so I hope it won't be too much of a hassle during implementation.

Thanks for the information.

**Comment by GerbilSoft** [comment]
Reopening this issue as a general "add more UDF stuff" issue.

**Comment by GerbilSoft** [comment]
Just now realized that I already had an issue open for UDF, #233. Closing this as Duplicate.
